### PR TITLE
Editorial changes in the spec

### PR DESCRIPTION
**Pull request by koto**
Resolving existing issues, and adding some more.

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -1334,14 +1325,12 @@ For the {{Element/insertAdjacentHTML()}} method, execute the
 [$Enforce a Trusted Type algorithm$] with input argument set to the *text*
 parameter value.
 
-Issue: do we need to specify that type argument should be {{TrustedHTML}}
-twice herein.
-
```
Comment (koto) [inline spec/index.bs]: Not really, this is implied from the sink definition (it accepts `HTMLString`, which is then converted in the algorithm to expectedType `TrustedHTML`)

**Comment by koto** [comment]
@mikesamuel can you review?

File: spec/index.bs [hunk 2]
```diff
@@ -1294,11 +1294,7 @@ At the beginning of step 5, insert the following steps:
 Note: See <a href="https://github.com/tc39/ecma262/issues/938">TC39/ecma262 issue #938</a>
 (adding a string to be compiled to algorithm parameters)
 
-Issue: Should we be modifying
-[[HTML5#hostensurecancompilestrings(callerrealm,-calleerealm)|HostEnsureCanCompileStrings]]
-or EnsureCSPDoesNotBlockStringCompilation?
-
-Modify [[CSP#can-compile-strings|HostEnsureCanCompileStrings]]
+Modify [[HTML5/webappapis#section-hostensurecancompilestrings|HostEnsureCanCompileStrings]]
```
Comment (mikesamuel) [inline spec/index.bs]: IIRC, this HTML abstract-op just calls through to the CSP one, so it seems less likely to cause problems if the change is made deeper.

File: spec/index.bs [hunk 3]
```diff
@@ -530,12 +530,10 @@ and exposed flag (*exposed*), run these steps:
     with the key *policyName*, return null.
 1.  Otherwise, let *policy* be the value of the *policyName* entry of
     *factory*'s `[[PolicyMap]]`.
-1.  If *policy*'s {{TrustedTypePolicyOptions/exposed|exposed}} attribute is false,
+1.  If *policy*'s `[[options]]` {{TrustedTypePolicyOptions/exposed|exposed}} attribute is false,
```
Comment (mikesamuel) [inline spec/index.bs]: 's after options?

File: spec/index.bs (hunk 2 above)
Comment (koto) [reply spec/index.bs]: The issue is that I want to have less integration points (to later sync the amendment to other specs), and I already integrate with HTML, so let's keep it like this.

**Pull request merged**